server/Dockerfile

```diff
@@ -0,0 +1,76 @@
+###################################
+# STAGE 1 - Install Server
+###################################
+FROM python:3.8-slim-buster as server
+# Setup variables
+ENV GUNICORN_DIRECTORY=/server/gunicorn \
+    GUNICORN_WORKERS=3 \
+    GUNICORN_WORKER_CLASS=gevent \
+    GUNICORN_BACKLOG=2048 \
+    GUNICORN_BIND=0.0.0.0:80 \
+    SERVER_MODULE=main:app \
+    SERVER_DIRECTORY=/server \
+    CLIENT_DIRECTORY=/client
+
+# Setup gunicorn config
+WORKDIR ${GUNICORN_DIRECTORY}
+COPY server/gunicorn .
+
+# Navigate to server directory
+WORKDIR ${SERVER_DIRECTORY}
+
+# Install server dependencies
+RUN pip install pipenv
+COPY server/Pipfile* ./
+RUN pipenv install --system --deploy
+
+# Copy remaining server source
+COPY server .
+
+
+###################################
+# STAGE 2 - Development
+###################################
+FROM server as development
```

Comment:
For development, none of the client code is needed, it will be accessed via the node development server.